tests/functional/adapter/utils/test_utils.py

```diff
@@ -33,6 +37,22 @@ class TestAnyValue(BaseAnyValue):
     pass
 
 
+# Not working because of https://github.com/trinodb/trino/issues/13
+@pytest.mark.skip(reason="No way to concat an array with null or empty array")
```

Comment:
The test is failing for the specific use case 

```
select 2 as id, {{ array_append(array_construct([]), 4) }} as array_col
```


Let's rather override the test case logic and omit this specific use case.